Backpack-SwiftUI/ImageGalleryGrid/Classes/ImageGalleryChipGrid.swift

```diff
@@ -0,0 +1,123 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+struct ImageGalleryChipGrid<ImageView: View>: ViewModifier {
+    let categories: [BPKImageGalleryCategoryChip<ImageView>]
+    let closeAccessibilityLabel: String
+    let onCategoryChanged: (BPKImageGalleryCategoryChip<ImageView>) -> Void
+    let onItemTapped: (BPKImageGalleryCategoryChip<ImageView>, BPKImageGalleryImage<ImageView>) -> Void
```

Comment:
why send a chip here? what the user taps is an Image in this case right? maybe this should be just the category and image?

Reply:
I think i'd also rename this to `onImgeTapped` to be clearer on what's being tapped

Reply:
So the 2 argument given here are in fact like you say the category and the image. Hopefully this will be clearer after the renaming I've done from your comment above "Please move this struct inside the enum of the Style above. same for the Image struct below"